### DISCUSSION TITLE
Documentation for Sample codes.

### DISCUSSION THREAD

**Discussion by KudaChan** [AvaloniaUI/Avalonia#14368] [Ideas]
Hello, community,

I like to invest my time and document the sample code for a clear understanding. I am new to AvaloniaUI and UI/UX and want to learn. Still, when I come to look at sample code to find a solution for my problem, I cannot understand anything due to pure code and no documentation. So, I have to read the entire code to find a solution, and if I document what I find in the code will help new and budding developers to learn Avalonia fast and help the community.

I like to know the community's opinion on this.

**Comment by workgroupengineering** [comment]
https://docs.avaloniaui.net/docs/welcome

https://github.com/AvaloniaUI/Avalonia.Samples

**Comment by thevortexcloud** [comment]
If you want to add docs for things, this list is a good place to start.

https://github.com/AvaloniaUI/avalonia-docs/issues/20

**Comment by timunie** [comment]
@KudaChan you can also get in touch with us on Telegram https://t.me/Avalonia
